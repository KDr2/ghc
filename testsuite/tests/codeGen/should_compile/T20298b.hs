module T20298b where

foo :: Integer
foo = 10 + 20
