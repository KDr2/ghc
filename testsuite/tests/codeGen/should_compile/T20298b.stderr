
==================== Tidy Core ====================
Result size of Tidy Core
  = {terms: 10, types: 3, coercions: 0, joins: 0/0}

foo2 = IS 10#

foo1 = IS 20#

foo = integerAdd foo2 foo1



